### ISSUE TITLE
Scrollable table of contents in knitted document?

### ISSUE THREAD

**Issue by megaraptor1** [juba/rmdformats#110]
This may be a feature request more than a particular issue, but I was wondering if there was a way to create a true scrollable table of contents using the `rmdformats` package? The floating table of contents is nice but I have noticed that if I have a document with a large number of levels to the table of contents (for example, the document I am working on now has at least 15 first level headings) I cannot easily get to the headings at the end of the document without using "ctrl+end" and then scrolling up to the correct section.

This might be more of a knitr issue in general, but I was wondering if there was any way to add a manual scroll bar to the table of contents if one wants to jump to the very end sections of a document?

**Comment by juba** [comment]
I guess you're still talking about the `html_clean` template ?
You're right, I never had a TOC long enough to see the problem, but this should now be fixed : the table of contents should become scrollable if its height is higher than the viewport.
Let me know if it doesn't work for you !

**Comment by megaraptor1** [comment]
Yes.

I reinstalled the package and it created a scroll bar. However, I noticed that it really cramped the table of contents. Indeed, I noticed that knitting the file produced a significant amount of white space not occupied by any text or figures. Is there any way to stretch things out so the width of the body and the table of contents remains the same as before prior to the scroll bar, but merely move the scroll bar to the side? I assume there is some command to do this.

![rmdformats example2](https://user-images.githubusercontent.com/74024763/121088416-baae7180-c7b3-11eb-9143-fb0bf332d638.jpg)
